tags/tag-developer-experience/charter.md

```diff
@@ -1 +1,67 @@
+
+# Scope
+
+TAG: Developer Experience (DevEx) focuses on enhancing the end-to-end experience of developers building cloud native applications. Our scope encompasses key concerns across various areas, including microservices, databases, streaming and messaging systems, AI, API management, and developer programming languages and frameworks, in addition to traditional developer experience.
+We explore and guide on:
+
+* Cloud Native application excellence
+* Inner/outer loop best practices
+* Observability and debugging from a developer’s perspective
+* Usability, DevEX research, and DevEX metrics across CNCF projects
+* AI development-related Developer Experience 
+
+Our work involves identifying gaps, promoting best practices, and supporting CNCF projects and communities to enhance the productivity and satisfaction of developers working with cloud native technologies. We aim to bridge tools, platforms, and practices to create a cohesive and frictionless developer experience across the cloud native ecosystem.
+
+More specifically, we:
+
+* Provide guidance and thought leadership on improving developer workflows and experience in cloud native application development
+* Promote secure-by-default developer workflows by integrating security and compliance considerations into the developer experience
+* Collaborate with CNCF projects to surface and resolve developer experience challenges
+* Conduct studies, gathering feedback from end users and contributors, and providing recommendations
+* Facilitate interoperability and standardization across developer tools in the CNCF ecosystem
+* Act as a bridge to empower developers and project maintainers
+
+# Out of Scope
+
+* Prescriptive mandates on tooling choices or project roadmaps
+* Evaluating or endorsing specific commercial developer tools outside of CNCF projects
+* Replacing or duplicating efforts by other TAGs unless explicitly partnered
+
+# Goals
+
+* Develop shared vocabulary, metrics, and frameworks to evaluate DevEX
+* Provide a central forum for discussing the developer experience within CNCF
+* Help CNCF projects better understand and serve developer needs
+* Publish guidance and reference materials for effective developer workflows
+* Encourage cross-project collaboration to improve consistency and interoperability of tooling
+* Foster collaboration around integrating security and compliance into cloud native development workflows to reduce friction and improve adoption of best practices
```

Comment:
Ditto as above, overlap with other specific groups

Reply:
Suggest (not specific to security and compliance):
In collaboration with other TAGs, integrate best practices into cloud-native development workflows to reduce friction and improve adoption.